_rules/word-spacing-not-important-9e45ec.md

```diff
@@ -67,6 +67,26 @@ This `strong` element has a `word-spacing` of `1.92pt !important` (equals `0.12e
 </html>
 ```
 
+#### Passed Example 2
+
+This `p` element has a [word-spacing][] of `12px` specified via the style attribute, which is above the recommended minimum given the specified font size is `1em`.
```

Comment:
The `p` should use relative units.

Reply:
I wanted to keep a few examples in non relative units. Is that ok?

Reply:
The suggestion to use relative units was to demonstrate good practice in our examples. If the font-size is using a relative unit, the word-spacing probably should as well.

Perhaps use fixed units for examples where the minimum requirements have been significantly exceeded.

Reply:
I think its nicer to have both units the same in each example.